### PR TITLE
Product Details

### PR DESCRIPTION
**Pull request by seansangpark**
Product Details ready for pull request prior to deployment.

### CODE REVIEW

File: client/src/productDetails/components/AddToCart.jsx [hunk 1]
```diff
@@ -15,42 +14,91 @@ const AddBagAndFavorite = styled.div`
...
+`;
+
+const QuantitySelector = styled.div`
+  width: 40%;
+  float: left;
+  padding: 10px;
+`;
+
+const CustomSelector = styled.select`
+  box-sizing: border-box;
```
Comment (juliebarwick) [inline client/src/productDetails/components/AddToCart.jsx]: You might not need this line anymore, not sure

File: client/src/productDetails/components/AddToCart.jsx [hunk 2]
```diff
@@ -87,13 +134,20 @@ const AddToCart = ({ styles, styleId, styleIndex, styleSelected, }) => {
   };
 
   const addToCart = () => {
-    // Adding to Cart: productName: , productId: , productStyle: , productStyleId: , price: ,
-    console.log(`size: ${selectedSize}, quantity: ${selectedQty}`);
+    const product = styles.product_id;
+    let style;
+    if (!styleIndex) {
+      style = styles.results[0].style_id
```
Comment (juliebarwick) [inline client/src/productDetails/components/AddToCart.jsx]: Maybe some semi-colons are missing?

**Review by juliebarwick — COMMENTED** [review]
A brief glance through - looks like some nice CSS changes. Sorry to not be more thorough though.

File: client/src/productDetails/components/AddToCart.jsx (hunk 1 above)
Comment (seansangpark) [reply client/src/productDetails/components/AddToCart.jsx]: Thank you!

File: client/src/productDetails/components/AddToCart.jsx (hunk 2 above)
Comment (seansangpark) [reply client/src/productDetails/components/AddToCart.jsx]: Good point! Issue resolved

**Pull request merged**